acs-integration-tests/src/test/java/com/ge/predix/test/utils/NewUaaTestUtil.java

```diff
@@ -0,0 +1,171 @@
+package com.ge.predix.test.utils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.oauth2.client.OAuth2RestTemplate;
+import org.springframework.security.oauth2.common.exceptions.InvalidClientException;
+import org.springframework.security.oauth2.provider.client.BaseClientDetails;
+import org.springframework.util.LinkedMultiValueMap;
+import org.springframework.util.MultiValueMap;
+
+import com.ge.predix.acs.utils.JsonUtils;
+
+public class NewUaaTestUtil {
+
+    private static final JsonUtils JSON_UTILS = new JsonUtils();
+
+    private final OAuth2RestTemplate adminRestTemplate;
+    private final String uaaUrl;
+    private final String tokenUrl;
+    private final ACSRestTemplateFactory acsRestTemplateFactory;
+
+    public NewUaaTestUtil(final OAuth2RestTemplate adminRestTemplate, final String uaaUrl) {
+        this.uaaUrl = uaaUrl;
+        this.tokenUrl = this.uaaUrl + "/oauth/token";
+        this.adminRestTemplate = adminRestTemplate;
+        this.acsRestTemplateFactory = null;
+    }
+
+    public NewUaaTestUtil(final ACSRestTemplateFactory acsRestTemplateFactory, final String uaaUrl) {
+        this.uaaUrl = uaaUrl;
+        this.tokenUrl = this.uaaUrl + "/oauth/token";
+        this.acsRestTemplateFactory = acsRestTemplateFactory;
+        this.adminRestTemplate = this.acsRestTemplateFactory.getOAuth2RestTemplateForUaaAdmin(this.tokenUrl);
+    }
+
+    public void createAcsAdminClient(final List<String> acsZones, final String clientId, final String clientSecret) {
+
+        BaseClientDetails acsAdminClient = new BaseClientDetails();
+        ArrayList<SimpleGrantedAuthority> authorities = new ArrayList<>();
+        authorities.add(new SimpleGrantedAuthority("acs.zones.admin"));
+        authorities.add(new SimpleGrantedAuthority("acs.attributes.read"));
+        authorities.add(new SimpleGrantedAuthority("acs.attributes.write"));
+        authorities.add(new SimpleGrantedAuthority("acs.policies.read"));
+        authorities.add(new SimpleGrantedAuthority("acs.policies.write"));
+        for (String zone : acsZones) {
+            authorities.add(new SimpleGrantedAuthority("predix-acs.zones." + zone + ".admin"));
```

Comment:
I don't think this authority is needed to administer a zone.  ACS required an authority per each administrative operation, e.g., acs.attributes.write to administer attributes, acs.policies.write to administer policies.  There is no blanket admin authority.